media/css/firefox/whatsnew/whatsnew-133-newsletter.scss

```diff
@@ -0,0 +1,68 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import 'includes/base';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+@import 'includes/mofo-donate-cta';
+
+.mzp-l-columns > div {
```

Comment:
**Nit:** this selector feels a bit too broad? Can we tighten it's scope to only the main page content perhaps?